circle.yml

```diff
@@ -13,10 +31,13 @@ jobs:
           when: always
       - store_test_results:
           path: ~/junit
-  modules:
+  modules-jdbc-test:
     steps:
       - checkout
-      - run: ./gradlew check -x testcontainers:check -x selenium:check
+      - run:
+          command: ./gradlew jdbc-test:check
```

Comment:
@bsideup I've split the CI build further so we now have four concurrent build jobs. I'm just wondering if I've missed anything: you'd _excluded_ certain check tasks, whereas I've gone on an _inclusive_ basis. This looks OK to me in local builds (i.e. it's compiling but not running tests on the core dependency, which seems correct).